vm/src/utils.rs

```diff
@@ -155,8 +158,9 @@ pub mod test_utils {
             }
         };
     }
-    pub(crate) use memory_from_memory;
+    pub use memory_from_memory;
```

Comment:
This is an auxiliary macro used to implement the `memory` macro, it shouldn't be made public

Reply:
These macros need to be public to make the `memory` and `segments` macros usable. Users need to import them as well.